### ISSUE TITLE
install broken on debian bullseye and possibly other platforms

### ISSUE THREAD

**Issue by glawler** [ory/kratos#566]
**Describe the bug**

Following the quickstart guide, I am unable to install kratos. 
Running `bash <(curl https://raw.githubusercontent.com/ory/kratos/master/install.sh) -b . v0.4.3-alpha.1` fails. 

**Reproducing the bug**

1. `docker run -it --rm debian:bullseye bash`
2. `apt update &&  apt install -y curl`

Then run the installer as instructed here: https://www.ory.sh/kratos/docs/install

```
root@30bdd94d587d:/# bash <(curl https://raw.githubusercontent.com/ory/kratos/master/install.sh) -b . v0.4.3-alpha.1
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 10303  100 10303    0     0   114k      0 --:--:-- --:--:-- --:--:--  114k
ory/kratos info checking GitHub for tag 'v0.4.3-alpha.1'
ory/kratos info found version: 0.4.3-alpha.1 for v0.4.3-alpha.1/linux/64bit
install: cannot stat '/tmp/tmp.j5ill1l0RJ/kratos': No such file or directory
root@30bdd94d587d:/# ls -ltr /tmp/tmp.j5ill1l0RJ/
total 39584
-rw-r--r-- 1 root root     1403 Jul  8 12:40 SECURITY.md
-rw-r--r-- 1 root root    16439 Jul  8 12:40 README.md
-rw-r--r-- 1 root root    11357 Jul  8 12:40 LICENSE
-rw-r--r-- 1 root root   102177 Jul  8 12:40 CHANGELOG.md
-rwxr-xr-x 1 root root 29769536 Jul  8 12:43 kratos-sqlite
-rw-r--r-- 1 root root 10618036 Jul  9 22:20 kratos_0.4.3-alpha.1-sqlite_linux_64bit.tar.gz
-rw-r--r-- 1 root root     2177 Jul  9 22:20 kratos_0.4.3-alpha.1_checksums.txt
root@30bdd94d587d:/# tar tvf /tmp/tmp.j5ill1l0RJ/kratos_0.4.3-alpha.1-sqlite_linux_64bit.tar.gz                            
-rw-r--r-- root/root    102177 2020-07-08 12:40 CHANGELOG.md
-rw-r--r-- root/root     11357 2020-07-08 12:40 LICENSE
-rw-r--r-- root/root     16439 2020-07-08 12:40 README.md
-rw-r--r-- root/root      1403 2020-07-08 12:40 SECURITY.md
-rwxr-xr-x root/root  29769536 2020-07-08 12:43 kratos-sqlite
root@30bdd94d587d:/# 
```
Looks like the binary that is untarred is `kratos-sqlite` but the install is looking for `kratos`

**Expected behavior**

The install,sh script will download and untar the `kratos` binary. And the install,sh script itself has some odd things in it like:

```
linux/amd64) BINARIES="kratos kratos kratos" ;;
```
As the binary to install. It wants to install `kratos` three times.

This is odd as well `NAME=${PROJECT_NAME}_${VERSION}-sqlite_${OS}_${ARCH}` which is forcing the `sqlite` build to be downloaded. 

**Environment**

- Version: v0.4.3-alpha.1
- Environment: Debian Bullseye in Docker

**Comment by aeneasr** [comment]
Yup, this has to do with how we name the binaries. Working on a fix which will hopefully be available by next week!

**Comment by glawler** [comment]
Nice. Thanks for the quick response.

**Comment by aeneasr** [comment]
This is resolved with 0.4.6 - thank you for raising this detailed issue!
